payload/Library/installapplications/installapplications.py

```diff
@@ -638,6 +651,9 @@ def main():
                         time.sleep(1)
 
```

Comment:
Couple of thoughts here

1. If you want to consider the "runtime start" to consider download and execution, you could just put `time.time()` above all of the types and reduce the code and variables.
2. Did you mean to include download as part of runtime?

Reply:
Depending on the answer, the time.time() could go on L627

Reply:
I like this idea, and I agree, it would reduce the number of changes/code.

In my mind, the download time should be included as part of the total runtime. This could aid administrators looking at logs seeing that something is taking a long while to download, etc.